src/DIRAC/WorkloadManagementSystem/DB/ElasticPilotParametersDB.py

```diff
@@ -0,0 +1,179 @@
+""" Module containing a front-end to the ElasticSearch-based ElasticPilotParametersDB.
+
+    The following class methods are provided for public usage
+      - getPilotParameters()
+      - setPilotParameter()
+      - deletePilotParameters()
+"""
+from DIRAC import S_OK, S_ERROR, gConfig
+from DIRAC.Core.Utilities import TimeUtilities
+from DIRAC.ConfigurationSystem.Client.PathFinder import getDatabaseSection
+from DIRAC.ConfigurationSystem.Client.Helpers import CSGlobals
+from DIRAC.Core.Base.ElasticDB import ElasticDB
+
+
+mapping = {
+    "properties": {
+        "PilotID": {"type": "long"},
+        "timestamp": {"type": "date"},
+        "CPUNormalizationFactor": {"type": "long"},
+        "Wallclock": {"type": "long"},
+        "HostName": {"type": "keyword"},
+        "GridCE": {"type": "keyword"},
+        "ModelName": {"type": "keyword"},
+        "Status": {"type": "keyword"},
+    }
+}
+
+
+class ElasticPilotParametersDB(ElasticDB):
+    def __init__(self, parentLogger=None):
+        """Standard Constructor"""
+
+        try:
+            section = getDatabaseSection("WorkloadManagement/ElasticPilotParametersDB")
+            indexPrefix = gConfig.getValue(f"{section}/IndexPrefix", CSGlobals.getSetup()).lower()
```

Comment:
Does this need to be configurable? Even if it does, it definitely shouldn't contain the setup.